2022-04-18 20:29:51,794 INFO    MainThread:237957 [wandb_setup.py:_flush():75] Loading settings from /home/alexzuzow/.config/wandb/settings
2022-04-18 20:29:51,795 INFO    MainThread:237957 [wandb_setup.py:_flush():75] Loading settings from /home/alexzuzow/Desktop/agar_multiagent/wandb/settings
2022-04-18 20:29:51,795 INFO    MainThread:237957 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-18 20:29:51,795 INFO    MainThread:237957 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'train_DQN.py', 'program': '/home/alexzuzow/Desktop/agar_multiagent/train_DQN.py'}
2022-04-18 20:29:51,795 INFO    MainThread:237957 [wandb_init.py:_log_setup():438] Logging user logs to /home/alexzuzow/Desktop/agar_multiagent/wandb/run-20220418_202951-3vzm26l3/logs/debug.log
2022-04-18 20:29:51,795 INFO    MainThread:237957 [wandb_init.py:_log_setup():439] Logging internal logs to /home/alexzuzow/Desktop/agar_multiagent/wandb/run-20220418_202951-3vzm26l3/logs/debug-internal.log
2022-04-18 20:29:51,796 INFO    MainThread:237957 [wandb_init.py:init():472] calling init triggers
2022-04-18 20:29:51,796 INFO    MainThread:237957 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {}
2022-04-18 20:29:51,796 INFO    MainThread:237957 [wandb_init.py:init():525] starting backend
2022-04-18 20:29:51,796 INFO    MainThread:237957 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-18 20:29:51,824 INFO    MainThread:237957 [backend.py:ensure_launched():219] starting backend process...
2022-04-18 20:29:51,846 INFO    MainThread:237957 [backend.py:ensure_launched():224] started backend process with pid: 238115
2022-04-18 20:29:51,847 INFO    MainThread:237957 [wandb_init.py:init():534] backend started and connected
2022-04-18 20:29:51,853 INFO    MainThread:237957 [wandb_init.py:init():598] updated telemetry
2022-04-18 20:29:51,902 INFO    MainThread:237957 [wandb_init.py:init():629] communicating run to backend with 30 second timeout
2022-04-18 20:29:52,455 INFO    MainThread:237957 [wandb_run.py:_on_init():1922] communicating current version
2022-04-18 20:29:52,509 INFO    MainThread:237957 [wandb_run.py:_on_init():1926] got version response 
2022-04-18 20:29:52,509 INFO    MainThread:237957 [wandb_init.py:init():660] starting run threads in backend
2022-04-18 20:29:54,556 WARNING MainThread:237957 [wandb_init.py:init():1017] interrupted
Traceback (most recent call last):
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 996, in init
    run = wi.init()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 671, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 439, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 225, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
